common/logisticspipes/blocks/powertile/LogisticsPowerJunctionTileEntity.java

```diff
@@ -121,6 +123,29 @@ public boolean canReceive() {
 		}
 	};
 
+	@ModDependentField(modId = "buildcraftlib")
+	private IMjReceiver mjReceiver = new IMjReceiver() {
```

Comment:
This is going to crash, when buildcraft is not installed

Reply:
No, because the field is never accessed when BC is not there, and IMjReceiver class is never loaded because of the annotation

Reply:
The ModDependentField only removes the Field not the Constructor call to initialise it

Reply:
Oh, you were right, it does crash with that error. Thought I tested that.